# `frame_id` of the header must be the sender's personal identifier

Header header
float64 bid
